src/Exception/ReceiverDoesNotExistException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KaroIO\MessengerMonitorBundle\Exception;
+
+use Symfony\Component\Messenger\Exception\RuntimeException;
+
+/**
+ * @internal
+ */
+class ReceiverDoesNotExistException extends RuntimeException
```

Comment:
We should consider adding `final` to any classes that we can. And if there are a few that we would like to make final, but cannot because we're mocking them, add an `@final` annotation instead.

Reply:
haha i HAVE phpstorm configured to add it automatically, but IIRC few years ago you told me that you don't do that in symfony (or you USED to not to that)
so, i methodically removed them! :joy:

Reply:
whoooops :D 

We may not have then... who knows. We're not "absolute" about it. But because it "minimizes" the API (especially when applied to non-internal classes), I like to add it.